etl-routes.js

```diff
@@ -242,28 +242,38 @@ module.exports = function () {
             }
         }, {
             method: 'GET',
-            path: '/etl/location/{uuid}/patient-flow-data',
+            path: '/etl/patient-flow-data',
             config: {
                 auth: 'simple',
                 plugins: {
                     'hapiAuthorization': { role: privileges.canViewClinicDashBoard }
                 },
                 handler: function (request, reply) {
-                    dao.getIdsByUuidAsyc('amrs.location', 'location_id', 'uuid', request.paramsArray[0],
-                        function (results) {
-                            request.paramsArray[0] = results;
+                    var asyncRequests = 0; //this should be the number of async requests needed before they are triggered
```

Comment:
@nkimaina don't you think this code should be in the DAO layer? This file should simply acts a facade (very thin)